streampipes-extensions/streampipes-processors-transformation-jvm/src/test/java/org/apache/streampipes/processors/transformation/jvm/processor/booloperator/counter/TestBooleanCounterProcessor.java

```diff
@@ -132,15 +122,22 @@ public void disconnect() throws SpRuntimeException {}
     Integer counter = sendEvents(booleanCounter, spOut);
```

Comment:
I'm getting an error on lines 121-122

```
booleanCounter.onPipelineStarted(params, spOut, null);
Integer counter = sendEvents(booleanCounter, spOut);
```

Params, spOut, booleanCounter. There's an error in this part